x/gov/simulation/genesis.go

```diff
@@ -17,12 +17,14 @@ import (
 
 // Simulation parameter constants
 const (
-	DepositParamsMinDeposit    = "deposit_params_min_deposit"
-	DepositParamsDepositPeriod = "deposit_params_deposit_period"
-	VotingParamsVotingPeriod   = "voting_params_voting_period"
-	TallyParamsQuorum          = "tally_params_quorum"
-	TallyParamsThreshold       = "tally_params_threshold"
-	TallyParamsVeto            = "tally_params_veto"
+	DepositParamsMinDeposit              = "deposit_params_min_deposit"
+	DepositParamsDepositPeriod           = "deposit_params_deposit_period"
+	VotingParamsVotingPeriodParamsChange = "voting_params_voting_period_params_change"
```

Comment:
Probably need to do search to replace, this is still using old naming for example

Reply:
as well other places

Reply:
indeed I did an other pass [c980a5c](https://github.com/atomone-hub/govgen/pull/16/commits/c980a5c5099770b76e7a6c6fc040399f82240510)